cypress/e2e/awx/views/schedules.cy.ts

```diff
@@ -109,34 +110,195 @@ describe('Schedules', () => {
   });
 });
 
-describe('Schedules - Create', () => {
-  it.skip('can create a simple schedule and navigate to the schedule details page', () => {
-    //Make sure to assert schedule summary, rules preview, exceptions preview
-    //Survey, prompts, rules and exceptions are not needed for this test
+describe('Schedules - Create', function () {
+  let organization: Organization;
+  let jobTemplate: JobTemplate;
+  let project: Project;
+  let inventory: Inventory;
+  let inventorySource: InventorySource;
+  before(() => {
+    cy.awxLogin();
+  });
+  beforeEach(function () {
+    project = this.globalProject as Project;
+    this.globalInventory;
+    cy.createAwxOrganization().then((o) => {
+      organization = o;
+      cy.createAwxInventory({ organization: organization.id }).then((i) => {
+        inventory = i;
+        cy.createAwxInventorySource(i, project).then((invSrc) => {
+          inventorySource = invSrc;
+        });
+      });
+    });
+    cy.navigateTo('awx', 'schedules');
```

Comment:
an `afterEach` is missing to delete the schedules created on these tests. Also the organizations, inventories, inventories sources, projects, job templated, etc created for these tests should be deleted as well.